### DISCUSSION TITLE
Issue in demultiplexing with only one hash usage

### DISCUSSION THREAD

**Discussion by Nagaemory** [satijalab/seurat#6459] [General]
Hi, 

only one hash used for an antigen in my sample and I couldn't do demultiplexing to know which cells are bind with that antigen. Any idea how to deal with this? Is there anyway to use the existing programs in Seurat?

Thanks

**Comment by rschauner** [comment]
So it sounds like you labelled all of your cells with an antibody and want to find out which cells were labelled. If that is the case, you can follow [this vignette on multimodal data](https://satijalab.org/seurat/articles/multimodal_vignette.html#setup-a-seurat-object-add-the-rna-and-protein-data-1). From there, you can either subset or label your cells which are bound to the antibody.

Cell hashing is used to multiplex multiple samples in one well of a 10x chip, this inherently requires more than one hashing antibody.
